mono/mini/xdebug.c

```diff
@@ -339,6 +339,10 @@ mono_save_xdebug_info (MonoCompile *cfg)
 void
 mono_save_trampoline_xdebug_info (MonoTrampInfo *info)
 {
+	const char *info_name = info->name;
+	if (info_name == NULL)
```

Comment:
Not sure as to "why" but there are numerous calls in the code to mono_tramp_info_create() with NULL in the first (name) argument.

Specifically in my case, the call seems to have originated at:
https://github.com/mono/mono/blob/master/mono/mini/mini-amd64.c#L8020

The lldb support code has a guard against NULL trampoline name, in the form of an empty string, hence a similar fix here.